packages/sdk/typescript/human-protocol-sdk/src/graphql/queries/operator.ts

```diff
@@ -26,22 +26,32 @@ const LEADER_FRAGMENT = gql`
 `;
 
 export const GET_LEADERS_QUERY = (filter: ILeadersFilter) => {
-  const { role } = filter;
+  const { role, roles, minAmountStaked } = filter;
 
   const WHERE_CLAUSE = `
     where: {
-      ${role ? `role: $role` : ''}
+      ${minAmountStaked ? `amountStaked_gt: $minAmountStaked` : ''}
```

Comment:
Should be including end?
```suggestion
      ${minAmountStaked ? `amountStaked_gte: $minAmountStaked` : ''}
```